app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -290,6 +302,18 @@ private fun assertAboutFirefoxPreview(): ViewInteraction {
         .check(matches(isCompletelyDisplayed()))
 }
 
+private fun assertAdvancedHeading() {
+    scrollToElementByText("Add-ons")
```

Comment:
Refer to one of my example code I recently changed where instead of this function I use the recycler view, and scroll within it. I think this sometimes fails if the element is off-screen (scrollToElementByText)